handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -1882,7 +1882,7 @@ protected ByteBuf compose(ByteBufAllocator alloc, ByteBuf cumulation, ByteBuf ne
                 return composite;
             }
             return attemptCopyToCumulation(cumulation, next, wrapDataSize) ? cumulation :
-                    composeIntoComposite(alloc, cumulation, next);
+                    copyAndCompose(alloc, cumulation, next);
```

Comment:
@Scottmitch I wonder if we want to make this configurable... Does this make any sense ?

Reply:
It is possible that folks may not want to copy under some situations (e.g. if each packet is over a certain threshold), but I'm not sure if providing this level of control is justified just yet. I propose we wait until it is necessary until we increase the API footprint. I think we could do something like `BiPredicate<ByteBuf, ByteBuf>` to determine if we should copy or use a Composite when the time comes.  WDYT?

Reply:
Yep fine as it is